bpf/max_tail_call.c

```diff
@@ -20,7 +20,7 @@
         return -1;                                       \
     }
 
-#define DECLARE_XDP_TAIL_FUNC(x) int xdp_test_callee##x(struct xdp_md* ctx);
```

Comment:
nit: should the functions be renamed to `sockops_...` since we are using sockops now?